src/DropDownMenu/DropDownMenu.js

```diff
@@ -364,6 +370,7 @@ class DropDownMenu extends Component {
             {displayValue}
           </div>
           <IconButton
+            disableTouchRipple={disableTouchRipple === undefined ? disabled : disableTouchRipple}
             tabIndex={this.props.disabled ? -1 : undefined}
```

Comment:
I don't think that we need that once we forward the `disabled` property as it's already inside the `EnhancedButton` component.